src/OpenLoco/Input/Keyboard.cpp

```diff
@@ -500,6 +500,7 @@ namespace OpenLoco::Input
                 catch (const std::exception&)
                 {
                     Windows::showError(StringIds::screenshot_failed);
+                    Ui::showMessageBox("Exception:", "screenshot_failed");
```

Comment:
If an error box is being shown we don't need an additional system error box. This is for situations where we can't use the normal error box